web/pages/[[...slug]].tsx

```diff
@@ -18,107 +18,75 @@ import { Slug } from '../types/Slug';
 import { Section } from '../types/Section';
 import { Hero as HeroProps } from '../types/Hero';
 import { mainEventId } from '../util/entityPaths';
+import {
+  ARTICLE_SECTION,
+  FIGURE,
+  HERO,
+  PROGRAM,
+  QUESTION_AND_ANSWER_COLLECTION_SECTION,
+  TEXT_AND_IMAGE_SECTION,
+  TICKET,
+} from '../util/queries';
 import styles from './app.module.css';
 
+const SHARED_SECTIONS = `
```

Comment:
Is this query specific to <em>[[...slug]].tsx</em> or could it also be moved to _../util/queries_?

Reply:
I wasn't sure what was best there. So far it's specific to [[...slug]].tsx though so are many of the others, so it's a gray area. Let's move it if more document types appear later on that also use `sections`, maybe.

(If it's moved to util/queries we'd have to decide if it should take event ID as a parameter or if that file should fetch `mainEventId` itself, I suppose.)